quick_classifier.py

```diff
@@ -69,7 +97,7 @@ def bottom_k_scores(scores, k):
 
     if args.records:
         records_path = args.records
-        out_path = records_path.replace('.csv', 'uat_keywords.tsv')
+        out_path = records_path.replace('.csv', '_uat_keywords.tsv')
```

Comment:
Make this file name configurable.

Reply:
Made output filename configurable in config.py.  Also, added command line arguments, `-p` or `--preserve_filename`, to preserve naming based on input filename.